src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
```

Comment:
Don't create instances of Random or other objects inside methods if they can be reused. Consider moving the instantiation of Random to a class-level field to avoid creating a new instance every time the method is called.